vmdb/app/presenters/menu/polymorph_item.rb

```diff
@@ -0,0 +1,7 @@
+module Menu
+  class PolymorphItem < Item
+    def name
+      super.call
```

Comment:
Still doesn't explain why you need a lambda.  A normal method can do the exact same thing, right?